.docker/app/Dockerfile

```diff
@@ -21,14 +21,16 @@ RUN apk add -U --no-cache \
       sqlite-dev=3.34.1-r0 \
       nodejs=14.16.1-r1 \
       yarn=1.22.10-r0 && \
-    gem update --system && \
     gem install bundler:2.2.17 && \
     bundle config build.nokogiri --use-system-libraries && \
-    bundle install && \
-    yarn install --check-files && \
-    bundle exec rails assets:precompile && \
-    yarn cache clean && \
+    bundle config set --local without "development test" && \
+    bundle install --jobs=8 && \
+    find "$GEM_HOME" -name Gemfile.lock -exec rm "{}" \; && \
+    find "$GEM_HOME" -name yarn.lock -exec rm "{}" \; && \
+    RAILS_ENV=production bundle exec rails assets:precompile && \
+    rm -rf node_modules && \
     rm -rf tmp/* && \
+    apk del build-dependencies && \
     rm -rf /var/cache/apk/ && \
     chown nobody /app/config/master.key && \
```

Comment:
In our other applications we handle this through an environment variable.  That means we don't have to worry about the existence of the file.  This works as long as the file is present, but I'm not sure how that would work in Rancher.  This threw an error for me locally because I didn't have that file.